var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -112,16 +126,20 @@ class Podio(CMakePackage):
         description="Use the specified C++ standard when building.",
     )
     variant("sio", default=False, description="Build the SIO I/O backend")
+    variant("rntuple", default=False, description="Build the RNTuple backend")
 
     # cpack config throws an error on some systems
     patch("cpack.patch", when="@:0.10.0")
     patch("dictloading.patch", when="@0.10.0")
     patch("python-tests.patch", when="@:0.14.0")
 
     depends_on("root@6.08.06: cxxstd=17", when="cxxstd=17")
-    depends_on("root@6.25.02: cxxstd=20", when="cxxstd=20")
+    depends_on("root@6.28.02: cxxstd=20", when="cxxstd=20")
```

Comment:
@wdconinc I suppose the `25` was a typo here and it should really be `6.28.02` for c++20 support from root. At least AFAIK, that was the one were it was officially announced.

Reply:
I can't recall why that version was there (maybe earliest c++20 support for the parts of root that podio uses?), so this change is fine.

In some sense it makes more sense to let root define this, and then it would be 6.28.04 since event 6.28.02 conflicts with cxxstd=20, https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/root/package.py#L324.

Reply:
Ah then maybe even `depends_on("root cxxstd=20, when="cxxstd=20")` here would be the best. I will have another look to see if I can simplify the dependencies and variants a bit further here.